### DISCUSSION TITLE
Cannot run watchface?

### DISCUSSION THREAD

**Discussion by s2-ieuan** [zepp-health/zeppos-forum#186] [Developer Feedback]
I have a GTR 2e, so I have to run the watchface maker in that mode.   

I can put my watchface together, but then when I want to run it on my device it just keeps spinning:  
![image](https://github.com/zepp-health/zeppos-forum/assets/126822343/456157a2-b016-4770-9896-52ea8cc552dd)

My dev tools show an error with ID 7764:  
![image](https://github.com/zepp-health/zeppos-forum/assets/126822343/75c1fb73-4d07-4809-94cb-2c67bf3932c0)

When I run in Zepp OS mode I do get a QR code, but I can't do anything with that as my watch does not have Zepp OS

I've tried both Chrome and Firefox, with no luck

**Comment by s2-ieuan** [comment]
@MyoungXUE from other discussions it seems that you are the person I'd need for an answer about this, apologies for @ing you directly but any chance you could weigh in on this problem?

**Comment by MyoungXUE** [comment]
Hi, only devices with Zepp OS can use Zepp OS Watchfaces and Mini Programs, the GTR 2e is not on the list of supported devices.

For devices that support Zepp OS, please see https://docs.zepp.com/docs/reference/related-resources/device-list/.

**Reply by s2-ieuan** [reply]
Right, but I selected the correct device when creating my project:  
![image](https://github.com/zepp-health/zeppos-forum/assets/126822343/763c1a63-5ee0-4230-88c8-6654e0ddbd35)

The issue is that I don't get a QR code for running the watchface, it produces the error in my start post. If I select Zepp OS I do get a QR code (which I cannot do anything with, I just did this to see if it would at least give me a QR code), so the issue is specifically in the non-Zepp OS version.

What can I do to get a QR code from the non-Zepp OS watchface? I can click Run on Device as shown below, but then nothing happens and I just get the error I shared.

![CleanShot 2023-05-29 at 05 48 45](https://github.com/zepp-health/zeppos-forum/assets/126822343/6bcd9e61-0caa-4b6a-a86c-eedb1eb4769c)

**Reply by s2-ieuan** [reply]
I get the exact same error if I try to use the `Export the watch face package` option, so it seems the problem is in building the actual watchface on the server side

**Reply by xuegang-liu** [reply]
What is your watch face configuration and have you added the necessary digital or analog clock? @s2-ieuan

**Reply by s2-ieuan** [reply]
@Journey-zepp Seems like that fixed it, so thanks! Would have been helpful if there was an actual error in the interface saying so, but this solves my issue for now

**Reply by xuegang-liu** [reply]
@s2-ieuan Thank you very much for your advice and glad to have solved your problem.
